icu4c/source/data/unidata/generate.sh

```diff
@@ -23,8 +23,7 @@ rm $ICU_SRC/icu4c/source/common/propname_data.h
 rm $ICU_SRC/icu4c/source/common/*_props_data.h
 rm $ICU4C_DATA_IN/*.icu
 rm $ICU4C_DATA_IN/*.nrm
-# TODO: Back to deleting coll/*.icu once ICU4X data generation is fixed.
-# rm $ICU4C_DATA_IN/coll/*.icu
+rm $ICU4C_DATA_IN/coll/*.icu
```

Comment:
Please also remove the following two lines, redundant with this one, to bring this part back to how it was before.

Reply:
Fixed.